src/category_theory/limits/limits.lean

```diff
@@ -548,22 +548,22 @@ def colimit.desc (F : J ⥤ C) [has_colimit F] (c : cocone F) : colimit F ⟶ c.
 @[simp] lemma colimit.is_colimit_desc {F : J ⥤ C} [has_colimit F] (c : cocone F) :
   (colimit.is_colimit F).desc c = colimit.desc F c := rfl
 
-@[simp] lemma colimit.ι_desc {F : J ⥤ C} [has_colimit F] (c : cocone F) (j : J) :
+@[simp, reassoc] lemma colimit.ι_desc {F : J ⥤ C} [has_colimit F] (c : cocone F) (j : J) :
   colimit.ι F j ≫ colimit.desc F c = c.ι.app j :=
 is_colimit.fac _ c j
 
-/--
-We have lots of lemmas describing how to simplify `colimit.ι F j ≫ _`,
-and combined with `colimit.ext` we rely on these lemmas for many calculations.
+-- /--
```

Comment:
Maybe we could move these comments above the first use of `reassoc` in this file, as a local explanation?